.github/workflows/test-deploy.yml

```diff
@@ -120,7 +120,13 @@ jobs:
 
           echo ::set-output name=tags::"$(
             for registry in docker.io/{dvcorg,iterativeai} ghcr.io/iterative; do
-              [[ "${{ matrix.latest }}" == "true" ]] && echo "${registry}/cml:latest"
+              if [[ "${{ matrix.latest }}" == "true" ]]; then
```

Comment:
Is not this also related  with some issues that we had with nvidia not accesible? maybe they were using cml:latest?

Reply:
Most of the users' issues on this side are either because they forget to use the `-gpu` tags or because they forget to pass `--gpus all` on GitHub.

Reply:
🤷🏼‍♂️

Reply:
Perhaps worth a mention on #652